### DISCUSSION TITLE
Import "solcx" could not be resolved

### DISCUSSION THREAD

**Discussion by arshook24** [smartcontractkit/full-blockchain-solidity-course-py#586] [Q&A] [answered]
I've installed, uninstalled, and reinstalled py-solc-x multiple times on Windows and cannot get the "Import "solcx" could not be resolved" error to go away. I know Windows sometimes needs extra care taken with paths, but I don't know what to do. Anything would be greatly appreciated!

My code matches the course at 3:44:27 exactly.

```Python
from solcx import compile_standard

with open("./SimpleStorage.sol", "r") as file:
    simple_storage_file = file.read()

compiled_sol = compile_standard(
    {
        "langauge": "Solidity",
        "sources": {"SimpleStorage/sol": {"content": simple_storage_file}},
        "settings": {
            "outputSelection": {
                "*": {"*": ["abi", "metadata", "evm.bytecode", "evm.sourceMap"]}
            }
        },
    },
    solc_version="0.6.0",
)
```

**Comment by cromewar** [comment, marked as answer]
It seems you code is pretty much ok, did you selected the correct python interpreter on vscode (the one you have solcx installed)?

**Reply by arshook24** [reply]
Hello! I'm not sure. How would I go about checking that?

**Reply by cromewar** [reply]
On the bottom left side on vscode you'll see a Python indicator with the version you are using, you should select the one where you have installed the packages.

On the same console you installed solcx type python --version to have and idea which interpreter you should use.

**Reply by arshook24** [reply]
Amazing!!! Looks like because I installed python from the Microsoft app store it was unfindable as version 3.9.9 was stored in the WindowsApps folder. I selected the interpreter (3.9.6) in my Programs folder instead and it compiled.

Thank you!

**Reply by cromewar** [reply]
Feel free to make any other questions, I'm glad is solved now.

**Reply by arshook24** [reply]
Will do!

**Comment by spencergoff** [comment]
I resolved this issue by using the command python3 deploy.py instead of python deploy.py .

**Reply by arshook24** [reply]
Interesting. I'll give this a shot, as well.
